.github/workflows/pull-request-test-azure.yml

```
@@ -0,0 +1,184 @@
+      - name: Check existing renkubot comment
+        if: needs.check-deploy.outputs.pr-contains-string == 'true'
+        uses: peter-evans/find-comment@v3
+        id: findcomment
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: "RenkuBot"
+      - name: Create comment pre deploy
+        if: ${{ steps.findcomment.outputs.comment-id == 0 &&
+          needs.check-deploy.outputs.pr-contains-string == 'true' }}
+        uses: peter-evans/create-or-update-comment@v4
+        with:
+          token: ${{ secrets.RENKUBOT_GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            You can access the deployment of this PR at https://ci-renku-${{ github.event.number }}.dev.azure.renku.ch
+
+  cypress-acceptance-tests:
+    name: Cypress tests on Azure
+    needs: [check-deploy, deploy-pr]
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        tests:
+          [
+            anonymousNavigation,
+            dashboardV2,
+            groupBasics,
+            projectBasics,
+            projectResources,
+            searchEntities,
+            sessionBasics,
+          ]
+    steps:
+      - uses: SwissDataScienceCenter/renku-actions/test-renku-cypress@v1.18.0
+        if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true' && needs.check-deploy.outputs.test-enabled == 'true'
+        with:
+          e2e-folder: cypress/e2e/v2/
+          e2e-target: ${{ matrix.tests }}
+          kubernetes-cluster-fqdn: dev.azure.renku.ch
+          renku-reference: ${{ github.ref }}
+          renku-release: ci-renku-${{ github.event.number }}
+          test-user-password: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
+
+
+  deploy-string-no-custom-version:
+    name: Ensure no custom components Azure
+    needs: [check-deploy]
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Check deploy string
+        if: ${{
+          needs.check-deploy.outputs.pr-contains-string == 'true'
+          && startsWith(github.base_ref, 'release-')
+          && (
+          needs.check-deploy.outputs.renku-core != null
+          || needs.check-deploy.outputs.renku-gateway != null
+          || needs.check-deploy.outputs.renku-graph != null
+          || needs.check-deploy.outputs.renku-notebooks != null
+          || needs.check-deploy.outputs.renku-ui != null
+          || needs.check-deploy.outputs.renku-data-services != null
+          )
+          }}
+        uses: actions/github-script@v7
+        with:
+          script: core.setFailed('Cannot merge release PR if it still has custom versions in deploy string.')
+  cleanup:
+    name: Cleanup Azure
+    if: github.event.action == 'closed'
+    runs-on: ubuntu-24.04
+    steps:
+      - name: renku teardown
+        uses: SwissDataScienceCenter/renku-actions/cleanup-renku-ci-deployments@v1.15.1
```

Everything else is on 1.18.0 is there a reason why this isnt?

```suggestion
        uses: SwissDataScienceCenter/renku-actions/cleanup-renku-ci-deployments@v1.18.0
```

---

Thanks, that had been changed since I created the first branch